<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="unique_name">
     <property name="text">
      <string>Unique Name:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="nickname">
     <property name="text">
      <string>Nickname:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="EncodingList" name="encoding_list">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="metadata_widget" native="true"/>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EncodingList</class>
   <extends>QWidget</extends>
   <header>brainframe.client.ui.dialogs.identity_configuration.encoding_list.encoding_list</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>encoding_list</sender>
   <signal>delete_encoding_class_signal(QString)</signal>
   <receiver>Form</receiver>
   <slot>delete_encoding_class_slot(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>110</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>75</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>delete_encoding_class_slot(QString)</slot>
 </slots>
</ui>
